helpers/installcredprovider.sh

```diff
@@ -13,7 +13,7 @@ RELEASEURL="$GITHUB/$REPO/releases/$VERSION/download/$FILE"
 NUGET_PLUGIN_DIR="$HOME/.nuget/plugins"
 
 # Ensure plugin directory exists
-if [[ ! -e "${NUGET_PLUGIN_DIR}" ]]; then
+if [ ! -d "${NUGET_PLUGIN_DIR/}" ]; then
```

Comment:
This slash isn't quite doing what the linked SO question was suggesting. It may even be trying to include the / in the name of the variable. We don't really care if the dir is a symlink in this case, so the extra / is unnecessary. If it's a symlink to a regular file, mkdir will fail, but that should be pretty rare and self-explanatory.

```suggestion
if [ ! -d "${NUGET_PLUGIN_DIR}" ]; then
```